src/app/auth/signup/signup.component.ts

```diff
@@ -41,7 +42,9 @@ export class SignupComponent implements OnInit {
           this.userData = form.value;
           this.userCheckData = this.userArray.find(loginUser => loginUser.userId === this.userData.userId )
           if(this.userCheckData){
-            alert('UserID already exists could you please check?')
+            const dialogref = this.dialog.open(DialogErrorComponent,{
```

Comment:
is the constant really needed?

Reply:
Yes as it is mentioned in material documentation.

Reply:
but i don't see it being reused anywhere? or can u point me where the constant is reused?